content/posts/backlogged-pinball.md

```diff
@@ -0,0 +1,99 @@
++++
+title =  "Backlogged Pinball"
+tags = ["pinball", "gcp", "next"]
+date = "2024-10-07"
++++
+
+# Backlogged Pinball
+
+Earlier this year, I built a pinball-based demo for [Cloud Next 2024](https://cloud.withgoogle.com/next).  I'm preparing some posts about specific aspects of it, but I wanted to post a brief overview first.  Backlogged is a physical pinball machine that has custom software to connect it to Cloud
+services.
+
+
+{{< figure src="/images/backlogged-pinball/pinball_suit.jpeg" width="600px" title="A conference attendee playing the Backlogged pinball demo" >}}
+
+## Demo overview
+
+
+
+Backlogged pinball was designed to demonstrate specific cloud products and
+benefits in a fun, interactive way.
+
+{{< figure src="/images/backlogged-pinball/backlogged-arch.png" width="600px" title="The pinball machine sends pubsub events to the cloud which are processed by Cloud Run and displayed on an interactive web app. Users can deploy their own Cloud Run service to process events and send pubsub messages back to the machine." >}}
+
+In its current iteration, it highlights:
+
+1.  Cloud PubSub: Two-way PubSub communication with a novel source of events
```

Comment:
Nitpick: (not a blocker) Consider updating PubSub instances to Pub/Sub.
```suggestion
1.  Cloud Pub/Sub: Two-way Pub/Sub communication with a novel source of events
```

Reply:
Done.